airflow/models.py

```diff
@@ -3678,7 +3757,7 @@ def refresh_from_db(self, session=None):
             DR.dag_id == self.dag_id,
             DR.execution_date == self.execution_date,
             DR.run_id == self.run_id
-        ).one()
+        ).first()
```

Comment:
I know. Why did you change it? Did you check all other code whether it is expecting a Exception or a None?